modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/ItemSelectorImpl.java

```diff
@@ -287,26 +287,27 @@ protected void activate(BundleContext bundleContext) {
 						bundleContext.ungetService(serviceReference);
 					}
 				});
-		_serviceTrackerMap = ServiceTrackerMapFactory.openMultiValueMap(
-			bundleContext, ItemSelectorViewRendererCustomizer.class, null,
-			ServiceReferenceMapperFactory.create(
-				bundleContext,
-				(itemSelectorViewRendererCustomizer, emitter) -> {
-					for (Class<? extends ItemSelectorCriterion>
-							itemSelectorCriterionClass :
-								itemSelectorViewRendererCustomizer.
-									getSupportedItemSelectorCriterionClasses()) {
-
-						emitter.emit(itemSelectorCriterionClass.getName());
-					}
-				}));
+		_itemSelectorViewRendererCustomizersServiceTracker =
```

Comment:
@W2jx Should be named `_xxxServiceTrackerMap`.

Reply:
I will fix it.